storybook/stories/API/cartesian/Line.stories.tsx

```diff
@@ -2,59 +2,121 @@
 import React from 'react';
 import { within } from '@storybook/testing-library';
 import { expect } from '@storybook/jest';
-import { Surface, Line, ResponsiveContainer } from '../../../../src';
-import { coordinateWithValueData } from '../../data';
+import { Surface, Line, ResponsiveContainer, ComposedChart } from '../../../../src';
+import { ccoordinateWithNullY, coordinateData, coordinateWithValueData } from '../../data';
 
 export default {
   component: Line,
   argTypes: {
+    connectNulls: {
+      control: {
+        type: 'boolean',
+      },
+    },
     stroke: {
       control: { type: 'color' },
     },
     fill: {
       control: { type: 'color' },
     },
+    type: {
+      // TODO: Generate from type instead.
```

Comment:
```suggestion
      // TODO: These options should be generated from the type directly instead of duplicating the type information here. Will iterate.
```